fineract-provider/src/main/java/org/apache/fineract/commands/domain/CommandWrapper.java

```diff
@@ -38,6 +38,8 @@
     private final String json;
     private final String transactionId;
     private final Long productId;
+    private final Boolean status;
```

Comment:
THe status field is basically, the status of the address. 
I could not find any better way to pass the field( which is a command) from API resource to command handler.